packages/commonwealth/client/styles/pages/notification_settings/index.scss

```diff
@@ -112,7 +112,7 @@
   }
 
   .chain-events-section-margin {
-    margin: 0 0 16px 0;
+    margin: 30px 0 16px 0;
```

Comment:
I noticed this section's margin did not match the others on the page, so fixing it here